osu.Game.Tests/Beatmaps/Formats/LegacyBeatmapEncoderTest.cs

```diff
@@ -40,13 +40,13 @@ public class LegacyBeatmapEncoderTest
         public void TestEncodeDecodeStability(string name)
         {
             var decoded = decodeFromLegacy(beatmaps_resource_store.GetStream(name), name);
-            var decodedAfterEncode = decodeFromLegacy(encodeToLegacy(decoded), name);
+            var (beatmap, beatmapSkin) = decodeFromLegacy(encodeToLegacy(decoded), name);
```

Comment:
arguably worse than before

Reply:
Yeah acutually after I read the diff... I probably prefer to revert that and again disable the inspection